app/assets/javascripts/discourse/app/instance-initializers/localization.js

```diff
@@ -45,12 +44,5 @@ export default {
       key = key.replace(/^[a-z_]*js\./, "");
       I18n._compiledMFs[key] = value;
     }
-
-    bootbox.addLocale(I18n.currentLocale(), {
-      OK: I18n.t("composer.modal_ok"),
-      CANCEL: I18n.t("composer.modal_cancel"),
```

Comment:
This was second to last use of this string. Maybe we can replace the last one (in insert-hyperlink modal) with the regular `cancel` string?